docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
+
+Linking internal `Group`s of users to their representations on LDAP servers allows for administrators to have an easier way to manage groups inside OpenShift. A command like `oadm sync groups` or `oadm sync group <name>` will trigger a sync of group members with their external support (if any).
+
+## Supporting Group Sources
+In order to link `Group` structures to external providers of their data, each `Group` will need an (optional) `GroupSource`:
```

Comment:
Be specific here.  There are a lot of different "users" and "groups" floating around.  Please qualify all uses with "openshift group", "openshift user", "openshift identity", "ldap group", "ldap user"